trophic_levels-biomass_indicators.rmd

```diff
@@ -102,28 +107,40 @@ This indicator alines most closely with B2 Structural state characteristics (alt
 
 ## Collinearities with other indicators
 
-Colinearities are likely to exist with climatic variables and disturbance. 
+Colinearities are likely to exist with climatic variables and disturbance.
 
 ## Reference condition and values
 
 ### Reference condition
 
-Reference values are taken from published global relationships between plants and vertebrate herbivores ([Fløjgaard et al. 2022](https://doi.org/10.1111/1365-2664.14047)) and between vertebrate prey and predators ([Hatton et al 2015](https://doi.org/10.1126/science.aac6284)). 
+Reference values are taken from published global relationships between plants and vertebrate herbivores ([Fløjgaard et al. 2022](https://doi.org/10.1111/1365-2664.14047)) and between vertebrate prey and predators ([Hatton et al 2015](https://doi.org/10.1126/science.aac6284)).
+
+For herbivores the expected biomass is calculated as:
+
+NPP^0.47^ \* 0.643
```

Comment:
Try like this:

$$
NPP^{0.47} \times 0.643
$$

Reply:
Not sure you can see the code in the equation above, but it's like this, only without the leading ':

'$$
'NPP^{0.47}\times0.643
'$$